src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -98,6 +98,18 @@ p5.RendererGL.prototype.vertex = function(x, y) {
     vertexColor[3]
   );
 
+  if (this.textureMode === constants.IMAGE) {
+    if (this.textureImage !== undefined) {
+      u /= this.textureImage.width;
+      v /= this.textureImage.height;
```

Comment:
do we need to be worried about divide by 0 here? or does something sensible enough happen / does some reasonable error appear in the console?

Reply:
I think divide by 0 returns the value Infinity. I can add in an additional check to make sure the width and height are > 0. Though I think if you have an image with a width and height of 0 you might have much bigger problems on your hands 😛

Reply:
totally, haha. I'm not sure what the right behavior is here; in some other places in the GL pipeline we do check if the divisor is 0 before running the division, though there's no error message printed out directly if that happens (see `camera()` for example)—so I'm not sure that feels right either! I wonder if it makes sense to come up with some guidelines here to make sure there's consistent behavior across places...